### PR TITLE
[excel] (Tutorials) Convert tutorial titles to reflect scenarios instead of features

### PR DESCRIPTION
**Pull request by AlexJerabek**
In an effort to drive tutorial usage, this PR adjusts the titles of the tutorial to focus on the task being achieved, rather than the skill being taught.

This PR also adjusts some of the language in the tutorials to focus on the reader, not the author ("you" instead of "we").

### CODE REVIEW

File: docs/develop/best-practices.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ A [`throw`](https://developer.mozilla.org/docs/web/javascript/reference/statemen
 
 However, if your script is running as part of a Power Automate flow, you may want to stop the flow from continuing. A `throw` statement stops the script and tells the flow to stop as well.
 
-The following script shows how to use the `throw` statement in our table checking example.
+The following script shows how to use the `throw` statement in the table checking example.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/best-practices.md:112]:
```suggestion
The following script shows how to use the `throw` statement in the table-checking example.
```

File: docs/tutorials/excel-power-automate-returns.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 ---
 
-# Return data from a script to an automatically-run Power Automate flow
+# Tutorial: Send weekly email reminders based on spreadsheet data
 
 This tutorial teaches you how to return information from an Office Script for Excel as part of an automated [Power Automate](https://make.powerautomate.com) workflow. You'll make a script that looks through a schedule and works with a flow to send reminder emails. This flow will run on a regular schedule, providing these reminders on your behalf.
 
 > [!TIP]
-> If you're new to Office Scripts, we recommend starting with the [Record, edit, and create Office Scripts in Excel](excel-tutorial.md) tutorial.
+> If you're new to Office Scripts, we recommend starting with the [Tutorial: Create and format an Excel table](excel-tutorial.md) tutorial.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-returns.md:13]: either remove "tutorial" from the end or from the reference title

```suggestion
> If you're new to Office Scripts, we recommend starting with [Tutorial: Create and format an Excel table](excel-tutorial.md).
```

File: docs/tutorials/excel-power-automate-returns.md [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 >
-> If you're new to Power Automate, we recommend starting with the [Call scripts from a manual Power Automate flow](excel-power-automate-manual.md) and [Pass data to scripts in an automatically-run Power Automate flow](excel-power-automate-trigger.md) tutorials.
+> If you're new to Power Automate, we recommend starting with the [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md) and [Tutorial: Automatically save content from emails in a workbook](excel-power-automate-trigger.md) tutorials.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-returns.md:15]:
```suggestion
> If you're new to Power Automate, we recommend starting with [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md) and [Tutorial: Automatically save content from emails in a workbook](excel-power-automate-trigger.md).
```

File: docs/develop/power-automate-integration.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ Power Automate lets you pass pieces of data between flow actions. Scripts can be
 
 Learn the details of how to pass data to and from your scripts with the following documentation.
 
-- Learn by doing with the [Pass data to scripts in an automatically-run Power Automate flow](../tutorials/excel-power-automate-trigger.md) and [Return data from a script to an automatically-run Power Automate flow](../tutorials/excel-power-automate-returns.md) tutorials.
+- Learn by doing with the [Tutorial: Automatically save content from emails in a workbook](../tutorials/excel-power-automate-trigger.md) and [Tutorial: Send weekly email reminders based on spreadsheet data](../tutorials/excel-power-automate-returns.md) tutorials.
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/power-automate-integration.md:71]:
```suggestion
- Learn by doing with [Tutorial: Automatically save content from emails in a workbook](../tutorials/excel-power-automate-trigger.md) and [Tutorial: Send weekly email reminders based on spreadsheet data](../tutorials/excel-power-automate-returns.md).
```

File: docs/develop/scripting-fundamentals.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ ms.localizationpriority: high
 
 This article will introduce you to the technical aspects of Office Scripts. You'll learn the critical parts of the TypeScript-based script code and how the Excel objects and APIs work together.
 
-If you would prefer to get started with an interactive experience, try the tutorial [Record, edit, and create Office Scripts in Excel](../tutorials/excel-tutorial.md) or visit our [samples](../resources/samples/samples-overview.md).
+If you would prefer to get started with an interactive experience, try the tutorial [Tutorial: Create and format an Excel table](../tutorials/excel-tutorial.md) or visit our [samples](../resources/samples/samples-overview.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/develop/scripting-fundamentals.md:13]:
```suggestion
If you would prefer to get started with an interactive experience, try [Tutorial: Create and format an Excel table](../tutorials/excel-tutorial.md) or visit our [samples](../resources/samples/samples-overview.md).
```

File: docs/resources/vba-differences.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ Office Scripts don't support Excel-level [events](/office/vba/excel/concepts/eve
 
 VBA doesn't have a Power Automate connector. All supported VBA scenarios involve a user attending to the macro's execution.
 
-Office Scripts can be run through Power Automate. Your workbook can be updated through scheduled or event-driven flows, letting you automate workflows without even opening Excel. Try the [Call scripts from a manual Power Automate flow](../tutorials/excel-power-automate-manual.md) tutorial to start learning about Power Automate. You can also check out the [Automated task reminders](scenarios/task-reminders.md) sample to see Office Scripts connected to Teams through Power Automate in a real-world scenario.
+Office Scripts can be run through Power Automate. Your workbook can be updated through scheduled or event-driven flows, letting you automate workflows without even opening Excel. Try the [Tutorial: Update a spreadsheet from a Power Automate flow](../tutorials/excel-power-automate-manual.md) tutorial to start learning about Power Automate. You can also check out the [Automated task reminders](scenarios/task-reminders.md) sample to see Office Scripts connected to Teams through Power Automate in a real-world scenario.
```
Comment (ElizabethSamuel-MSFT) [inline docs/resources/vba-differences.md:45]:
```suggestion
Office Scripts can be run through Power Automate. Your workbook can be updated through scheduled or event-driven flows, letting you automate workflows without even opening Excel. Try [Tutorial: Update a spreadsheet from a Power Automate flow](../tutorials/excel-power-automate-manual.md) to start learning about Power Automate. You can also check out the [Automated task reminders](scenarios/task-reminders.md) sample to see Office Scripts connected to Teams through Power Automate in a real-world scenario.
```

File: docs/tutorials/excel-power-automate-manual.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
 
 > [!TIP]
-> If you are new to Office Scripts, we recommend starting with the [Record, edit, and create Office Scripts in Excel](excel-tutorial.md) tutorial. [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
+> If you are new to Office Scripts, we recommend starting with the [Tutorial: Create and format an Excel table](excel-tutorial.md) tutorial. [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-manual.md:13]:
```suggestion
> If you are new to Office Scripts, we recommend starting with [Tutorial: Create and format an Excel table](excel-tutorial.md). [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```

File: docs/tutorials/excel-power-automate-manual.md [hunk 8]
```diff
@@ -125,4 +125,4 @@ Your flow is now ready to be run through Power Automate. You can test it using t
 
 ## Next steps
 
-Complete the [Pass data to scripts in an automatically-run Power Automate flow](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script and run the Power Automate flow when certain events occur.
+Complete the [Tutorial: Automatically save content from emails in a workbook](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script and run the Power Automate flow when certain events occur.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-manual.md:128]:
```suggestion
Complete [Tutorial: Automatically save content from emails in a workbook](excel-power-automate-trigger.md). It teaches you how to pass data from a workflow service to your Office Script and run the Power Automate flow when certain events occur.
```

File: docs/tutorials/excel-power-automate-trigger.md [hunk 9]
```diff
@@ -1,16 +1,16 @@
...
+> If you're new to Office Scripts, we recommend starting with the [Tutorial: Create and format an Excel table](excel-tutorial.md) tutorial. If you're new to Power Automate, we recommend starting with the [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md) tutorial. [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-trigger.md:13]:
```suggestion
> If you're new to Office Scripts, we recommend starting with [Tutorial: Create and format an Excel table](excel-tutorial.md). If you're new to Power Automate, we recommend starting with [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md). [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```

File: docs/tutorials/excel-power-automate-trigger.md [hunk 10]
```diff
@@ -30,7 +30,7 @@ Power Automate shouldn't use [relative references](../testing/power-automate-tro
 
     ```TypeScript
     function main(workbook: ExcelScript.Workbook) {
-      // Add a new worksheet to store our email table
+      // Add a new worksheet to store the email table
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-trigger.md:33]:
```suggestion
      // Add a new worksheet to store the email table.
```

File: docs/tutorials/excel-power-automate-trigger.md [hunk 11]
```diff
@@ -220,6 +220,6 @@ Receiving multiple emails at the same time can cause merge conflicts in Excel. T
 
 ## Next steps
 
-Complete the [Return data from a script to an automatically-run Power Automate flow](excel-power-automate-returns.md) tutorial. It teaches you how to return data from a script to the flow.
+Complete the [Tutorial: Send weekly email reminders based on spreadsheet data](excel-power-automate-returns.md) tutorial. It teaches you how to return data from a script to the flow.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-power-automate-trigger.md:223]:
```suggestion
Complete [Tutorial: Send weekly email reminders based on spreadsheet data](excel-power-automate-returns.md). It teaches you how to return data from a script to the flow.
```

File: docs/tutorials/excel-read-tutorial.md [hunk 12]
```diff
@@ -1,16 +1,16 @@
...
 
 > [!TIP]
-> If you're new to Office Scripts, we recommend starting with the [Record, edit, and create Office Scripts in Excel](excel-tutorial.md) tutorial. [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
+> If you're new to Office Scripts, we recommend starting with the [Tutorial: Create and format an Excel table](excel-tutorial.md) tutorial. [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-read-tutorial.md:13]:
```suggestion
> If you're new to Office Scripts, we recommend starting with [Tutorial: Create and format an Excel table](excel-tutorial.md). [Office Scripts use TypeScript](../overview/code-editor-environment.md) and this tutorial is intended for people with beginner to intermediate-level knowledge of JavaScript or TypeScript. If you're new to JavaScript, we recommend starting with the [Mozilla JavaScript tutorial](https://developer.mozilla.org/docs/Web/JavaScript/Guide/Introduction).
```

File: docs/tutorials/excel-read-tutorial.md [hunk 13]
```diff
@@ -144,4 +144,4 @@ Now that you know how to read and write to a single cell, you can generalize the
 
 Open the Code Editor and try out some of our [Sample scripts for Office Scripts in Excel](../resources/samples/excel-samples.md). You can also visit [Fundamentals for Office Scripts in Excel](../develop/scripting-fundamentals.md) to learn more about creating Office Scripts.
 
-The next series of Office Scripts tutorials focus on using Office Scripts with Power Automate. Learn more about the advantages combining the two platforms in [Run Office Scripts with Power Automate](../develop/power-automate-integration.md) or try the [Call scripts from a manual Power Automate flow](excel-power-automate-manual.md) tutorial to create a Power Automate flow that uses an Office Script.
+The next series of Office Scripts tutorials focus on using Office Scripts with Power Automate. Learn more about the advantages combining the two platforms in [Run Office Scripts with Power Automate](../develop/power-automate-integration.md) or try the [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md) tutorial to create a Power Automate flow that uses an Office Script.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-read-tutorial.md:147]:
```suggestion
The next series of Office Scripts tutorials focus on using Office Scripts with Power Automate. Learn more about the advantages combining the two platforms in [Run Office Scripts with Power Automate](../develop/power-automate-integration.md) or try [Tutorial: Update a spreadsheet from a Power Automate flow](excel-power-automate-manual.md) to create a Power Automate flow that uses an Office Script.
```

File: docs/tutorials/excel-tutorial.md [hunk 14]
```diff
@@ -109,9 +109,9 @@ Let's convert this fruit sales data into a table. We'll use our script for the e
...
 1. Create a new worksheet in the current workbook.
-2. Copy the fruit data from the beginning of the tutorial and paste it into the new worksheet, starting at cell **A1**.
-3. Run the script.
+1. Copy the fruit data from the beginning of the tutorial and paste it into the new worksheet, starting at cell **A1**.
+1. Run the script.
 
 ## Next steps
 
-Complete the [Read workbook data with Office Scripts in Excel](excel-read-tutorial.md) tutorial. It teaches you how to read data from a workbook with an Office Script.
+Complete the [Tutorial: Clean and normalize Excel workbook data](excel-read-tutorial.md) tutorial. It teaches you how to read data from a workbook with an Office Script.
```
Comment (ElizabethSamuel-MSFT) [inline docs/tutorials/excel-tutorial.md:117]:
```suggestion
Complete [Tutorial: Clean and normalize Excel workbook data](excel-read-tutorial.md). It teaches you how to read data from a workbook with an Office Script.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Now that the titles of the tutorials include the word "tutorial", consider removing "Tutorial:" from the link titles when they're referenced, or remove "tutorial" or "tutorial" that follow the links.

Sorry - it's raining "tutorial"...lol

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT! I debated between your suggestions and removing the "Tutorial:" from the links. Your way is less text, so we'll go with that :-)

**Pull request merged**